catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2146,6 +2146,9 @@ private QueryResponse doQuery(QueryRequest queryRequest, FederationStrategy stra
 
         LOGGER.debug("Calling strategy.federate()");
 
+        if (fanoutEnabled) {
+            queryRequest.getProperties().put("no-default-tags", true);
```

Comment:
Can we create constant for `no-default-tags` somewhere instead of hard-coding it?